src/components/Tables/Table.tsx

```diff
@@ -127,15 +127,15 @@ export type FilterConfig<Filter> =
           primaryFiltersShown: true;
           primaryFilters: Filter[];
           setPrimaryFilters: (primaryFilters: Filter[]) => void;
-          additionalFiltersShown: false;
+          additionalFiltersShown: true;
+          additionalFilters: Filter[];
+          setAdditionalFilters: (additionalFilters: Filter[]) => void;
       }
     | {
           primaryFiltersShown: true;
           primaryFilters: Filter[];
           setPrimaryFilters: (primaryFilters: Filter[]) => void;
-          additionalFiltersShown: true;
-          additionalFilters: Filter[];
-          setAdditionalFilters: (additionalFilters: Filter[]) => void;
+          additionalFiltersShown: false;
```

Comment:
It looks like you've just swapped these around, can we revert this to avoid spurious changes